packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkDecoding.ts

```diff
@@ -228,6 +234,29 @@ export class InlineArrayDecoder implements ChunkDecoder {
 	}
 }
 
+/**
+ * Decoder for {@link EncodedIncrementalChunkShape}s.
+ */
+export class IncrementalChunkDecoder implements ChunkDecoder {
+	public constructor(private readonly cache: DecoderContext<EncodedChunkShape>) {}
+	public decode(_: readonly ChunkDecoder[], stream: StreamCursor): TreeChunk {
+		const chunkReferenceId = readStream(stream);
+		assert(
+			typeof chunkReferenceId === "number",
+			"Expected incremental field's data to be a chunk reference id",
+		);
+
```

Comment:
You could use:
```suggestion
		const chunkReferenceId = readStreamNumber(stream);
```

That has the assert built in.